qiskit/transpiler/_transpilererror.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2018 IBM RESEARCH. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+
+"""
+Exception for errors raised by the transpiler.
+"""
+from qiskit import QISKitError
+
+
+class TranspilerError(QISKitError):
+    """Exceptions raised during transpilation"""
+    def __init__(self, *msg):
```

Comment:
If you're not overriding the behaviour of `QISKitError`, remove the body of the class completely. You don't need even `pass` the docstring is enough.

Reply:
ok. i copied from other custom errors, like `SimulatorError`. Those should probably be updated too.